services/slam/builtin/cartographer_int_test.go

```diff
@@ -18,6 +19,11 @@ import (
 	"go.viam.com/utils/artifact"
 )
 
+const (
+	cartoSleepMS = 100
+)
+
+// TODO DATA-916 remove lua files from tests
```

Comment:
[opt] TODO is probably not necessary, since this won't be hard for the implementer of DATA-916 to find.